source/diff/diff.cpp

```diff
@@ -375,13 +380,12 @@ class Differ {
   SpvExecutionModel GetExecutionModel(const opt::Module* module,
                                       uint32_t entry_point_id);
   std::string GetName(const IdInstructions& id_to, uint32_t id, bool* has_name);
-  std::string GetFunctionName(const IdInstructions& id_to, uint32_t id);
+  std::string GetSanitizedName(const IdInstructions& id_to, uint32_t id);
```

Comment:
You should add some comment that describe what this functions does.  "Sanitize" could mean a lot of different things.

Reply:
Done.

Reply:
(open to better name suggestions)